packages/mui-material/src/FormLabel/FormLabel.d.ts

```diff
@@ -7,7 +7,12 @@ import { FormLabelClasses } from './formLabelClasses';
 
 export interface FormLabelPropsColorOverrides {}
 
-export interface FormLabelOwnProps {
+/**
+ * This type is kept for compatibility. Use `FormLabelOwnProps` instead.
+ */
+export type FormLabelBaseProps = React.LabelHTMLAttributes<HTMLLabelElement>;
+
+export interface FormLabelOwnProps extends FormLabelBaseProps {
```

Comment:
Other `*OwnProps` interfaces don't contain native element's HTML attributes. Let's keep it that way.

```suggestion
export interface FormLabelOwnProps {
```

Reply:
fixed in `FormLabelTypeMap` by defining `props: AdditionalProps & FormLabelBaseProps & FormLabelOwnProps`